assets/store_descriptions/generate_and_validate.py

```diff
@@ -3,20 +3,27 @@
 import xml.etree.ElementTree as ET
 import glob
 import sys
+import os
 
 
 en_tree = ET.parse('assets/store_descriptions/en-US/strings.xml')
 en_root = en_tree.getroot()
 
 def getString(key):
     try:
-        string = root.findall(key)[0].text
+        string = root.findall(".//string[@name='" + key + "']")[0].text
     except:
-        string = en_root.findall(key)[0].text
+        string = en_root.findall(".//string[@name='" + key + "']")[0].text
+    return(string)
+
+def unescapeString(string):
+    string = string.replace('\\"', '"')
+    string = string.replace("\\'", "'")
     return(string)
 
 strings_files=glob.glob('assets/store_descriptions/*/strings.xml')
 for file in strings_files:
+    print(file)
     tree = ET.parse(file)
     root = tree.getroot()
     lang=file[26:-12]
```

Comment:
This can't be stable :) How about a regex?

Reply:
I was thinking about a regex as well, but something like `assets/store_descriptions/*/strings.xml` is as stable as the substring command I use.